### ISSUE TITLE
 black box testing

### ISSUE THREAD

**Issue by SongXY283** [jaewonalive/PeerAiD#1]
May I ask if you can provide a proxy model for black box testing? Thank you very much

**Comment by jaewonalive** [comment]
Hi, thank you for your interest in our work!

Here is the checkpoint for the surrogate model I used in Table 6 for black-box testing:

- **ResNet-34** : https://drive.google.com/file/d/144sGKDenTk31R5yfTqW6GhtE9cS0tc7-/view?usp=sharing
- **MobileNetV2** : https://drive.google.com/file/d/1mF8hLIh9u1H6Xe9MzioS8VU3bGGYfhpw/view?usp=sharing

I trained these two surrogate models with PGD-10 adversarial training, as described in the paper. 

Although I have already mentioned the training settings in the paper, I am providing them again here for your reference:

- **Optimizer**:

    -  SGD with momentum 0.9 and weight decay 0.0002

- **Learning rate**:

    - Initialized at 0.1 and decayed by a factor of 10 at 100 and 150 epochs

- **Batch size**:

    - 128

- **Training epochs**:

    - 200 epochs, with the best epoch selected based on the highest PGD-10 validation accuracy.

    - The best epoch was at 100, which occurred immediately after the first learning rate decay.

Thank you!
